pytlu/online_monitor/pytlu_producer_sim.py

```diff
@@ -0,0 +1,89 @@
+''' This is a producer faking data coming from pytlu by taking real data and sending these in chunks'''
```

Comment:
Not really a fix ;-) It is a nice feature to have to enable hardware less testing.